.github/workflows/build-gluon.yml

```diff
@@ -3,8 +3,8 @@ on:
   push:
     branches:
       - master
-      - next*
-      - v20*
+      - next
+      - 'v20[2-9][0-9].[0-9].x'
```

Comment:
My idea was to just have a single branch here, differing between the next, master and release branches, but I'm fine with this solution, too.

Reply:
This is the "cleaner" solution, however this adds one more task when branching.

As workflow run is limited to max. 1 for pushes on branches, it has not functional impact when keeping the references in all branches.